src/server.ts

```diff
@@ -1,7 +1,6 @@
 import Fastify from "fastify";
 import cors from "@fastify/cors";
-import { project, projects } from "./controllers";
-import editTicket from "./controllers/ticket";
+import { project, projects, ticket } from "./controllers";
```

Comment:
This line should fix: https://github.com/ryonryon/trello-clone-backend/pull/8/files#r865543959